cli/src/commands/tui/mod.rs

```diff
@@ -180,3 +189,28 @@ pub async fn start_ui(app: &Arc<Mutex<App>>) -> std::io::Result<bool> {
 
     Ok(true)
 }
+
+pub fn panic_hook(panic_info: &PanicInfo<'_>) {
+    let mut stdout = stdout();
+
+    let msg = match panic_info.payload().downcast_ref::<&'static str>() {
+        Some(s) => *s,
+        None => match panic_info.payload().downcast_ref::<String>() {
+            Some(s) => &s[..],
+            None => "Box<Any>",
```

Comment:
what does this mean?

Reply:
This was originally taken from [here](https://github.com/ClementTsang/bottom/blob/9d0bfbbf93f85907893fd879710eaba44160b236/src/lib.rs#L298)

As my understanding was that its a placeholder to match the string return types

Reply:
ok got it